### PR TITLE
Inherit labels for service and ingress

### PR DESCRIPTION
**Pull request by mpeters0n**

### CODE REVIEW

File: charts/lenses/templates/ingress.yaml [hunk 1]
```diff
@@ -11,6 +11,9 @@ metadata:
     release: {{ .Release.Name }}
     lenses.io/app: {{ include "fullname" . }}
     lenses.io/app.type: lenses-ingress
+    {{- range $key, $value := .Values.labels }}
+    {{ $key }}: {{ $value | quote }}
+    {{- end }}
```
Comment (mpeters0n) [inline charts/lenses/templates/ingress.yaml:16]: Inherit the top-level labels

File: charts/lenses/templates/service.yaml [hunk 2]
```diff
@@ -12,6 +12,9 @@ metadata:
     release: {{ .Release.Name }}
     lenses.io/app: {{ include "fullname" . }}
     lenses.io/app.type: lenses-service
+    {{- range $key, $value := .Values.labels }}
+    {{ $key }}: {{ $value | quote }}
+    {{- end }}
```
Comment (mpeters0n) [inline charts/lenses/templates/service.yaml:17]: Inherit the top-level labels

**Comment by mpeters0n** [comment]
I tested and verified that these small changes meet our requirements for cluster label enforcement

**Comment by georgeyord** [comment]
Thank you @mpeters0n for this addition! Lenses helm chart latest version is `4.1.5` is at your disposal and includes this change.

**Pull request merged**